### ISSUE TITLE
April 2018 Shipment for biological activity evaluation to Dundee

### ISSUE THREAD

**Issue by maratsydney** [OpenSourceMalaria/Series4#39]
## Update 25/7/18: Second run data has been received - [link](https://github.com/OpenSourceMalaria/Series4/issues/56)

OSM group in Sydney decided that there is no better way to celebrate Malaria day than send new batch of compounds for evaluation! Who knows, maybe the key molecule for the victory over Malaria now sitting in queue for  the plane to UK.

The second batch of compounds for year (shown below) have now been shipped to Dundee for screening against _P. falciparum_.

--------------------------------------------------------------------------------------------------
- Two batches of resynthesised OSM-S-106 (made by @mbhebhe)
- Miscellaneous compounds (L to R): Both enantiomers of aminoalcohol attached to the LHS with Boc-rotection and as free amines + Nemesis compounds #38 (made by @edwintse);  products of coupling with pyrazole + 4-chlorophenol ether on RHS (made by me); the "wrong" Pfizer phenol analogs ([#555](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/555)) (made by @david1597).
- New set of suzuki products inspired by recent active hit OSM-S-525 #31 including resynthesised OSM-S-525, 4-nitrophenyl compound OSM-S-553 (proposed by @MFernflower) and very promising compound OSM-S-557 that possess two very active substituents on both sides (made by me).
![2018 04 24 dundee](https://user-images.githubusercontent.com/29498087/39183900-e9bb001a-4804-11e8-9c09-70a238688644.png)
--------------------------------------------------------------------------------------------------
MMV Number | SMILES | InChI | InChI Key
-- | -- | -- | --
MMV025100 | NC1=C2C(C=C(C3=CC(S(N)(=O)=O)=CC=C3)S2)=NC=N1 | InChI=1S/C12H10N4O2S2/c13-12-11-9(15-6-16-12)5-10(19-11)7-2-1-3-8(4-7)20(14,17)18/h1-6H,(H2,13,15,16)(H2,14,17,18) | MQMXDJVOZKMSNT-UHFFFAOYSA-N
MMV1580421 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OCC(CO)C4=CC=CC(OCC5=CC=CC=C5)=C4)N32 | InChI=1S/C28H24F2N4O4/c29-28(30)38-23-11-9-20(10-12-23)27-33-32-25-14-31-15-26(34(25)27)37-18-22(16-35)21-7-4-8-24(13-21)36-17-19-5-2-1-3-6-19/h1-15,22,28,35H,16-18H2 | MNCMMCAPVFCUDR-UHFFFAOYSA-N
MMV1580422 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OCC(CO)C4=CC=CC(O)=C4)N32 | InChI=1S/C21H18F2N4O4/c22-21(23)31-17-6-4-13(5-7-17)20-26-25-18-9-24-10-19(27(18)20)30-12-15(11-28)14-2-1-3-16(29)8-14/h1-10,15,21,28-29H,11-12H2 | OKGRHDYGIKFDNY-UHFFFAOYSA-N
MMV1580423 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OC[C@H](NC(OC(C)(C)C)=O)CC4=CC=CC=C4)N32 | InChI=1S/C26H27F2N5O4/c1-26(2,3)37-25(34)30-19(13-17-7-5-4-6-8-17)16-35-22-15-29-14-21-31-32-23(33(21)22)18-9-11-20(12-10-18)36-24(27)28/h4-12,14-15,19,24H,13,16H2,1-3H3,(H,30,34) | SDPYEXOSVNFLTG-UHFFFAOYSA-N
MMV1580424 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OC[C@H](N)CC4=CC=CC=C4)N32 | InChI=1S/C21H19F2N5O2/c22-21(23)30-17-8-6-15(7-9-17)20-27-26-18-11-25-12-19(28(18)20)29-13-16(24)10-14-4-2-1-3-5-14/h1-9,11-12,16,21H,10,13,24H2/t16-/m1/s1 | FXPPGJNSMWQGRD-MRXNPFEDSA-N
MMV1580425 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OC[C@@H](NC(OC(C)(C)C)=O)CC4=CC=CC=C4)N32 | InChI=1S/C26H27F2N5O4/c1-26(2,3)37-25(34)30-19(13-17-7-5-4-6-8-17)16-35-22-15-29-14-21-31-32-23(33(21)22)18-9-11-20(12-10-18)36-24(27)28/h4-12,14-15,19,24H,13,16H2,1-3H3,(H,30,34) | SDPYEXOSVNFLTG-UHFFFAOYSA-N
MMV1580426 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OC[C@@H](N)CC4=CC=CC=C4)N32 | InChI=1S/C21H19F2N5O2/c22-21(23)30-17-8-6-15(7-9-17)20-27-26-18-11-25-12-19(28(18)20)29-13-16(24)10-14-4-2-1-3-5-14/h1-9,11-12,16,21H,10,13,24H2/t16-/m0/s1 | FXPPGJNSMWQGRD-INIZCTEOSA-N
MMV1580427 | FC1=CC(CCOC2=CN=CC3=NN=C(N4C=CC=N4)N32)=CC=C1F | InChI=1S/C16H12F2N6O/c17-12-3-2-11(8-13(12)18)4-7-25-15-10-19-9-14-21-22-16(24(14)15)23-6-1-5-20-23/h1-3,5-6,8-10H,4,7H2 | ACHYQCFQKMAIMF-UHFFFAOYSA-N
MMV1580428 | FC1=CC(CCOC2=NN=C3C=NC=C(N4C=CC=N4)N32)=CC=C1F | InChI=1S/C16H12F2N6O/c17-12-3-2-11(8-13(12)18)4-7-25-16-22-21-14-9-19-10-15(24(14)16)23-6-1-5-20-23/h1-3,5-6,8-10H,4,7H2 | NTTWNFXYVLPPAS-UHFFFAOYSA-N
MMV1580429 | FC(C=C1)=C(F)C=C1CCOC2=CN=CC3=NN=C(OC4=CC=C(Cl)C=C4)N32 | InChI=1S/C19H13ClF2N4O2/c20-13-2-4-14(5-3-13)28-19-25-24-17-10-23-11-18(26(17)19)27-8-7-12-1-6-15(21)16(22)9-12/h1-6,9-11H,7-8H2 | ZAKLZMUOIOUUEH-UHFFFAOYSA-N
MMV1580430 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC(N(C)C)=CC=C4)N32)=CC=C1F | InChI=1S/C21H19F2N5O/c1-27(2)16-5-3-4-15(11-16)21-26-25-19-12-24-13-20(28(19)21)29-9-8-14-6-7-17(22)18(23)10-14/h3-7,10-13H,8-9H2,1-2H3 | CHKSIGXRRUITRE-UHFFFAOYSA-N
MMV1580431 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C5C(NN=C5)=C4)N32)=CC=C1F | InChI=1S/C20H14F2N6O/c21-15-4-1-12(7-16(15)22)5-6-29-19-11-23-10-18-26-27-20(28(18)19)13-2-3-14-9-24-25-17(14)8-13/h1-4,7-11H,5-6H2,(H,24,25) | CGVAIQJJRYJQMC-UHFFFAOYSA-N
MMV1580432 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=C5C(NC=C5)=CC=C4)N23)=CC=C1F | InChI=1S/C21H15F2N5O/c22-16-5-4-13(10-17(16)23)7-9-29-20-12-24-11-19-26-27-21(28(19)20)15-2-1-3-18-14(15)6-8-25-18/h1-6,8,10-12,25H,7,9H2 | BAKHABOPVVJMJZ-UHFFFAOYSA-N
MMV1580433 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C(Cl)C(N)=C4)N32)=CC=C1F | InChI=1S/C19H14ClF2N5O/c20-13-3-2-12(8-16(13)23)19-26-25-17-9-24-10-18(27(17)19)28-6-5-11-1-4-14(21)15(22)7-11/h1-4,7-10H,5-6,23H2 | UUAZLIWHOSTWRP-UHFFFAOYSA-N
MMV1580434 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=CC(N)=C4)N32)=CC=C1F | InChI=1S/C19H15F2N5O/c20-15-5-4-12(8-16(15)21)6-7-27-18-11-23-10-17-24-25-19(26(17)18)13-2-1-3-14(22)9-13/h1-5,8-11H,6-7,22H2 | CQZOPOXKTPBXOH-UHFFFAOYSA-N
MMV1580435 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C(CC)C=C4)N23)=CC=C1F | InChI=1S/C21H18F2N4O/c1-2-14-3-6-16(7-4-14)21-26-25-19-12-24-13-20(27(19)21)28-10-9-15-5-8-17(22)18(23)11-15/h3-8,11-13H,2,9-10H2,1H3 | GYRZMHSUHLZDJA-UHFFFAOYSA-N
MMV1580436 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C(C(O)=O)C=C4)N32)=CC=C1F | InChI=1S/C20H14F2N4O3/c21-15-6-1-12(9-16(15)22)7-8-29-18-11-23-10-17-24-25-19(26(17)18)13-2-4-14(5-3-13)20(27)28/h1-6,9-11H,7-8H2,(H,27,28) | UMTJNRMURJHRAT-UHFFFAOYSA-N
MMV1580437 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=CC(C(O)=O)=C4)N32)=CC=C1F | InChI=1S/C20H14F2N4O3/c21-15-5-4-12(8-16(15)22)6-7-29-18-11-23-10-17-24-25-19(26(17)18)13-2-1-3-14(9-13)20(27)28/h1-5,8-11H,6-7H2,(H,27,28) | GYFXXYGBOJTVGJ-UHFFFAOYSA-N
MMV1580438 | FC1=CC(CCOC2=CN=CC3=NN=C(C4=CC=C([N+]([O-])=O)C=C4)N32)=CC=C1F | InChI=1S/C19H13F2N5O3/c20-15-6-1-12(9-16(15)21)7-8-29-18-11-22-10-17-23-24-19(25(17)18)13-2-4-14(5-3-13)26(27)28/h1-6,9-11H,7-8H2 | ANXKNNLWLQBISN-UHFFFAOYSA-N
MMV1580441 | OC(C1=CC=CC=C1)COC2=CN=CC3=NN=C(C4CNCCC4)N32 | InChI=1S/C18H21N5O2/c24-15(13-5-2-1-3-6-13)12-25-17-11-20-10-16-21-22-18(23(16)17)14-7-4-8-19-9-14/h1-3,5-6,10-11,14-15,19,24H,4,7-9,12H2 | SNDLBGDEOQPBOQ-UHFFFAOYSA-N
MMV1580442 | FC1=C(F)C=CC(CCOC2=CN=CC3=NN=C(C4CNCCC4)N32)=C1 | InChI=1S/C18H19F2N5O/c19-14-4-3-12(8-15(14)20)5-7-26-17-11-22-10-16-23-24-18(25(16)17)13-2-1-6-21-9-13/h3-4,8,10-11,13,21H,1-2,5-7,9H2 | DRTAQDBSMNPECX-UHFFFAOYSA-N
MMV1580315 | FC(F)OC(C=C1)=CC=C1C2=NN=C3C=NC=C(OCC(CO)(O)C4=CC=CC=C4)N32 | InChI=1S/C21H18F2N4O4/c22-20(23)31-16-8-6-14(7-9-16)19-26-25-17-10-24-11-18(27(17)19)30-13-21(29,12-28)15-4-2-1-3-5-15/h1-11,20,28-29H,12-13H2 | LVBNVRWXODMMAV-UHFFFAOYSA-N
MMV1580439 | OCC(COC1=CN=CC2=NN=C(C3=CC=C4C(NC=C4)=C3)N21)C5=CC=CC=C5 | InChI=1S/C22H19N5O2/c28-13-18(15-4-2-1-3-5-15)14-29-21-12-23-11-20-25-26-22(27(20)21)17-7-6-16-8-9-24-19(16)10-17/h1-12,18,24,28H,13-14H2 | XPFWGOXNOABZNA-UHFFFAOYSA-N

[2018.04.24 Dundee.cdx.zip](https://github.com/OpenSourceMalaria/Series4/files/1942459/2018.04.24.Dundee.cdx.zip)

**Comment by MFernflower** [comment]
I think i found a mistake in the dataset? @maratsydney 
![39183900-e9bb001a-4804-11e8-9c09-70a238688644](https://user-images.githubusercontent.com/3164942/39206807-6370c892-47cc-11e8-9d72-f608f8b7190d.png)

**Comment by maratsydney** [comment]
It is indeed unusual, I was aiming for the MK031-2a but apart from it isolated another product and with help of NMR was able to assign structure shown as MK031-2b.  ![mk031-2 github](https://user-images.githubusercontent.com/29498087/39219786-0eb54842-4870-11e8-9278-121d85966969.png)

**Comment by MFernflower** [comment]
Perhaps you had a mirrored impurity in your halogenated precursor?

**Comment by mattodd** [comment]
It's a super-interesting compound @maratsydney has carefully isolated. At the moment we can only think that this implies some lability in the ether substituent. i.e. That it can, under certain conditions, be displaced and can then re-attach. But it needs more investigating if we're going to understand it. @mcoster

**Comment by drc007** [comment]
@maratsydney Do you see any product where t-BuOH has displaced the Cl?

**Comment by mcoster** [comment]
Hmm, interesting... Are you able to share the NMR data - what evidence shows that it is this particular regioisomer?

**Comment by drc007** [comment]
Could it be? 
![idea](https://user-images.githubusercontent.com/7080516/39230960-5fff91c2-4860-11e8-8159-e4ca4766f42a.png)

**Comment by drc007** [comment]
@mcoster @maratsydney Might be difficult to distinguish by NMR? 

Is it a solid? Small molecule crystal structure.

**Comment by maratsydney** [comment]
@drc007, @mcoster definitely crystal will be the best option. Now after you mentioned that isomer that I didn't consider before I got confused a bit again (need to look back at all NMR), but one of the non-direct evidences for proposed structure is coupling of protons on main heterocycle which appear when I nave nitrogen in north-west position (and J value is quite big 4-5 Hz) while in all other compounds these signals sharp singlets. And I imagine in proposed structure coupling between protons of main heterocycle looks impossible (just too far away from each other). See all the spectra for two compounds in attachment in [MestreNova format](https://au-mynotebook.labarchives.com/share_attachment/Marat%2520Korsik/MjAyLjh8MjkzNzAvMTU2LTE4MDAvVHJlZU5vZGUvOTY1NDIxOTN8NTE0Ljg=).

**Comment by mcoster** [comment]
MK031-2a: 9.1 ppm seems quite far downfield for a pyrazine 1H signal - have you seen them this far downfield in other compounds?

**Comment by maratsydney** [comment]
@mcoster Just checked that for starting material (with bromine on the RHS) it's 8.9 ppm, for compound with naked phenyl ring - 9.0 ppm, with 4-nitrophenyl 9.1.
@drc007 I isolated only these two product in reasonable amounts, I have seen some very weak response during the run of the column, but it would be less than mg quantity - so I didn't try to analyze it.

**Comment by mattodd** [comment]
Interesting discussion and ideas here. @maratsydney I wonder if you can lay out the case (i.e. give us a pic of the NMR spectrum) for why we do not have the isomeric bromide present at the start of the displacement reaction. There is this issue of why sometimes we have singlets and sometimes doublets for the two protons attached to the pyrazine ring, but I'm pretty sure that's not related.

**Comment by mcoster** [comment]
@maratsydney - thanks for uploading the NMRs! I spent some time this morning poring over them, and I agree with you that it seems to be the transposed isomer, although a crystal structure would be fantastic. What a surprising result! It's interesting that no compound with pyrazole in both north-west and north-east is isolated. I wonder whether this chemistry could be optimised and then exploited to access analogues that are otherwise difficult to prepare?..

**Comment by maratsydney** [comment]
@mcoster thank you for spending time on NMR, always good to know that more experienced people support the same version. I am currently working towards getting crystals. And just to close that important point about the purity I attached NMR of that batch used in reaction. 

@mattodd  good point that you mentioned about these doublets that we see sometimes. I will try to organize it nicely and will open another issue to ask about that interesting case, since all my attempts to find the answer were unsuccessful so far.
[MK005-14.zip](https://github.com/OpenSourceMalaria/Series4/files/1957519/MK005-14.zip)
[MK005-14.pdf](https://github.com/OpenSourceMalaria/Series4/files/1957520/MK005-14.pdf)

**Comment by maratsydney** [comment]
Short but very important update:
I received results from x-ray analysis of mentioned above compounds and the structure for suggested MK031-2b proved to be wrong. Here is the reaction scheme with products+xray structures that explain H NMR perfectly but completely unexpected in terms of chemistry:
<img width="1125" alt="2018 05 22 - mk031-2 xray" src="https://user-images.githubusercontent.com/29498087/40354122-83451c96-5df6-11e8-8f12-1fa7ab8e8749.png">

Apart from it I have gt results of other x-ray samples that have doublets and they found to be connected the same way at the bottom of the core at position 8. Example below (NMR attached):


![mk026-2 scheme](https://user-images.githubusercontent.com/29498087/40354868-496a500c-5df8-11e8-8e4f-2542fb5778c3.png)
[MK026-2.zip](https://github.com/OpenSourceMalaria/Series4/files/2025995/MK026-2.zip)
After these news were revealed I looked at compounds that were made in the project so far and found that some of them have same characteristic doublets an in previous examples. Now were are trying to run more experiments to get clarity on this unexpected chemistry.
And the last point that triazolopyrazine with OH attached to it that was recovered as a side product by @david1597 has triazolopyrazine protons as doublets as well, that strongly suggest that actual position is 8 rather than 5. So please take this into considireation @Plebellec @CamilleHauguel @mcoster

**Comment by drc007** [comment]
@maratsydney Looks like a nice short publication there ;-)

**Comment by mcoster** [comment]
Well now, that's interesting! Great work @maratsydney

**Comment by drc007** [comment]
So do you think the displacement at the 8-position occurred when you introduced the difluorophenylethanol or as you have drawn it in the reaction above there was a rearrangement from the 5-positon to the 8-position when you introduced the pyrrazole?

**Comment by mattodd** [comment]
It's a fascinating and important result. Three things:

1) @maratsydney has a spreadsheet of instances he'll share where the doublets are seen, which suggest attachment at the 8- position. They are uniformly biologically inactive. These data may require structural revision of several compounds (annoying) but also may help explain why some minor structural changes resulted in inactivity, re-opening some SAR we thought we'd explored.

2) The most likely explanation is that the chlorine was in the 8-position all along. @maratsydney is double-checking this by looking back at data on the chlorinated precursors that were used. At the moment this analysis is suggesting that the chlorine was not in the 8-position, but was where we expected it to be.

3) If that's the case, and the chlorine was in the right place in the precursor (and there was, for example, no impurity of a compound with the chlorine in the 8-position) then we are left to explain how a migration, or some other rearrangement, could possibly have happened. Such a reaction would require not just the migration of the chain, but also the migration of the H. At the moment, Marat's analysis suggests that we see these products when the added nucleophile is nucleophilic at N or S, rather than O. An experiment to be performed, therefore, is to use one batch of chlorinated precursor in three reactions that are identical apart from the addition of i) an alcohol, ii) an analogous amine and iii) an analogous thiol.

**Comment by mcoster** [comment]
Here are my thoughts on a plausible mechanism for formation of the unusual regioisomer, if it turns out to not be from a contaminant in the starting materials:

![unusual-regioisomer_snar](https://user-images.githubusercontent.com/28282596/40404434-2dccaa36-5e9b-11e8-8d1a-cd1be2a145d5.png)

In the expected reaction pathway, the nucleophile attacks the _ipso_ carbon, bearing the leaving group. With some nucleophiles, perhaps based on hard-soft properties, pKa's, reversibility, proton sources, phase of the moon ;) - attack _para_ to the chlorine could give a [Meisenheimer complex](https://en.wikipedia.org/wiki/Meisenheimer_complex) that is protonated, then deprotonated at a different position to give a new Meisenheimer complex, which loses chloride to give the unexpected regioisomer.

(Notes: there are lots of resonance forms that could be drawn for the Meisenheimer complex, and where proton transfers occur, it is reasonable to invoke the nucleophile, ie. hydroxide/water, amine/ammonium salt, etc.)

**Comment by drc007** [comment]
I guess in the first step you park the anion on N7 then pick up the hydrogen at C5 to get to the same intermediate? Followed by simple elimination not involving triazole.

You could also take MK031-2a and resubject to reaction conditions to see if MK031-2b is formed?

**Comment by mcoster** [comment]
Yes, N7 anion will certainly be major resonance contributor

**Comment by mcoster** [comment]
The resonance contributor I drew does look uglier than the one with negative charge on N7, but they both contribute and I think overall, this resonance stabilised intermediate is as reasonable as any nucleophilic attack at C2 of a pyridine type system.

The final elimination could be concerted. I just drew it stepwise because the intermediate is analogous to the ipso-attack intermediate. I've drawn that with negative charge on the triazole nitrogen, but there are other resonance contributors with negative charge on the triazole carbon plus C6 and C8 of the pyrazine ring. 

It will be nicely consistent if C8 substitution is only seen with softer nucleophiles. At this stage, it looks like @CamilleHauguel 's reaction with hydroxide gave a product with singlets rather than doublets for the pyrazine protons.

**Comment by maratsydney** [comment]
Thanks everyone for comments. We are working in the group on systematic study of this interesting case and soon will be opened separate issue. So far all the data suggest rearrangement/migration of the group rather than impurity since it was seen for at least three different triazolopyrazine substrates that were used to make compounds that have singlets protons signals on pyrazine (more details on this will follow later). 

@mcoster I just checked ELN of @CamilleHauguel and I believe that this very same route as @david1597  used should deliver the same product.  Main point is to be aware of it.

**Comment by MedChemProf** [comment]
Just as external point of comparison for NMR spectra, there is a J.Med.Chem., **2017**, 60(13), pp. 5673-5698 (https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00397) that contains a good number of analogs with the Class 4 scaffold. This paper may have been shared before, but I thought it prudent to lost it again. The linker region of these analogs is much simpler than most of ours.

**Comment by MFernflower** [comment]
Wee bit unrelated but think we can get samples of the compounds they were
testing and send them off for malaria screening? @medchemprof @mattodd

>

**Comment by mcoster** [comment]
Great find @MedChemProf - it's interesting how streamlined their triazolopyrazine library synthesis is.

**Comment by MFernflower** [comment]
I wonder if this unexpected rearrangement reaction could make for a nice short paper to Nature?? Perhaps @mattodd can open a public draft of the paper like we are for s4 drug paper?

**Comment by edwintse** [comment]
Experiments are underway but I think we need to try to figure out what's going on first before we decide where to publish. (Might take longer that expected too).

**Comment by MFernflower** [comment]
Any word on results @mattodd

**Comment by maratsydney** [comment]
@MFernflower we are still waiting for results, will update as we receive data from Dundee.

**Comment by david1597** [comment]
Potency data for the first run of the April batch has been received:

![24042018 dundee results](https://user-images.githubusercontent.com/30584472/40752994-629d9dfe-64b5-11e8-9db1-16d0df3032cb.jpg)

- The original Suzuki hit (OSM-S-525) which was resynthesised has come back at 168 nM confirming good potency (86 nM average of two runs last time).
- Others from the Suzuki series are also potent. A quick look seems to indicate that an NH meta, and something para gives the more potent compounds.
- The 2-phenylpropan-1-ol compounds show some promise - potencies are in the low hundred nM.
- Nemesis is inactive
- Phenylalinaninol compounds are pretty much inactive but show variation between enantiomers

Once we have data from the second run (usually 7 days wait) I will repost and then update the wiki/master list/ELN/GH etc. We're still waiting on the Series 3 result, and that of the Pfizer re-run that was sent separately. 

Great to see two sets of potent compounds in this selection.

**Comment by MFernflower** [comment]
@david1597 Wonderful! I guess this allows the green light to tinker even more with the 2-phenylpropan-1-ol series 

A few possible ideas: 
![april2018](https://user-images.githubusercontent.com/3164942/40753958-945ce816-6444-11e8-822f-b8053506646a.png)

**Comment by david1597** [comment]
FYI, here are the compounds we plan to send in the next shipment (potentially in the next 1-2 weeks). It may answer some questions about what we should do next, esp regarding the Suzuki series.

![compounds for dundee](https://user-images.githubusercontent.com/30584472/40753993-21187ade-64ba-11e8-9151-e5b6be9e35ea.jpg)

**Comment by edwintse** [comment]
@MFernflower It's usually a good idea to provide some rationale for your suggestions. It'll help when making decisions on what to synthesise next. (e.g. any particular reason for the tricyclic or biphenyl ideas?)

**Comment by MFernflower** [comment]
@edwintse The idea behind the biphenyl was because of the potency of the benzyl protected molecule (OSM-S-532) and the tricyclic is simply a ring lock of OSM-S-532

**Comment by mcoster** [comment]
How exciting! I would echo @MedChemProf 's suggestion in #49 that we need to keep our target compound profile in mind. There seem to be a number of recent structural changes that benefit potency, eg. benzylic OH, RHS indole/meta-aniline, meta-bezyloxy substituted LHS. However, we also need to keep in mind what these changes do to the ADME/DMPK properties. If we start a new issue in the near future to prioritise future directions, it would be really helpful to chart out the key structures, with not only the potency, but also cLogP and other key indicators. For example, the meta-benzyloxy LHS compund is interesting, but I presume it is higher logP, lower water solubility, etc? Might suggest priority should be given to exploration of the RHS indoles and indole isosteres.

**Comment by MFernflower** [comment]
Sadly most of the compounds tested and the ones proposed by me are pretty oily - What do you think of the proposed amide? It is predicted to sub three logP but I am concerned about the amide group being quickly metabolized 
![swissadme](https://user-images.githubusercontent.com/3164942/40755822-0aff2534-644e-11e8-9286-536e9b8f351a.png)

**Comment by mattodd** [comment]
Agreed @mcoster - just commented in response to @MedChemProf at https://github.com/OpenSourceMalaria/Series4/issues/49#issuecomment-393367988 The attractive aspect of the Pfizer and indole/meta-NH2 compounds is that the potency is achieved through the addition of polar groups. I've not calculated, but I'm assuming the cLogPs are good, and we need to maintain this trend for all future compounds here. @MedChemProf proposed benzylic amines, for example, are great and maintain the other polar groups. Also avoiding metabolically labile groups (or toxic) groups is then the remaining item on the checklist.

**Comment by MFernflower** [comment]
@drc007 @mattodd Indeed - all focus should be put into the palladium coupling and other steps needed to get the 5-phenethylamino compound but I do feel at some point we should combine the things we have learned about the RHS and LHS into a single molecule (Addressing the rather polar elephant in the room so to speak)

![elephant](https://user-images.githubusercontent.com/3164942/40756322-e2201dd2-6450-11e8-90df-5f31839e2388.png)

**Comment by mcoster** [comment]
One thing to bear in mind if the meta-amino substituted compounds are explored further - many anilines have nasty metabolites, including mutagenic properties. [Here's a paper](https://pubs.acs.org/doi/pdf/10.1021/jm3001295) from AZ where they looked at getting rid of mutagenicity in aminobiphenyls (admittedly, the para-substituted ones, though). They have some interesting observations around putting a fluorine or chlorine meta to the amine, so it is also ortho- to the other aryl group - there's an interesting interplay between the inductive effect of the halogen and the conformational change around the biaryl bond.

The above, and looking at the 3D models we made at GU as trophies for undergrad prizes has me wondering about substitution at this position of the RHS substituent. I can't recall having seen this explored in the compounds tested previously? Changing the orientation of the RHS substituent relative to the triazolopyrazine core could have an interesting influence on the activity, particularly if there is a H-bonding group, eg. indoles, meta-amines.

![biaryl-angle](https://user-images.githubusercontent.com/28282596/40757364-79479c7e-64cb-11e8-8b72-46af53a853cd.png)

**Comment by mattodd** [comment]
[Tolerated, for sure, but need something para](https://github.com/OpenSourceMalaria/Series4/wiki/Modification-of-the-Triazole-Substitution).

**Comment by drc007** [comment]
A few thoughts. Some very interesting results but perhaps we need to get repeats before embarking on long syntheses ;-)
We should remember that as @mcoster has mentioned anilines have a well deserved reputation for being mutagenic. In addition chloroanilines are known to be hepatotoxic, and putative aromatic ring hydroxylation metabolites are often more toxic. If we plan to follow this lead up then it would be prudent to have a strategy in place to assess the potential toxicity risk. 
I would suspect there is an intramolecular hydrogen bond between the NH2 and the ortho chlorine, so perhaps the "virtual ring" is is acting as a indole mimic? It might be interesting to replace the 4-chloro with 4-Methyl (or ethyl) to compare.
@david1597 The N-Methyl indole that is in the next batch could be very informative (I assume the lack of double bond in the structure is a typo?).
@mcoster I'd always assumed the aryl and triazolopyrazine groups were orthogonal in the low energy conformation but with relatively free rotation. If we introduce substituents (your X or Y) will we get atropisomers?

**Comment by mattodd** [comment]
Yes indeed - we need to wait for the repeats from Dundee (usually 7 days) and the usual thing (hopefully this batch is the same, otherwise we'll chase) is to secure cytotox data on actives.

**Comment by david1597** [comment]
@drc007 Typo. I've edited the original to fix it. Thanks

**Comment by mcoster** [comment]
@drc007 that "virtual ring" idea is interesting - the fluoro analogue on @david1597 's list will be interesting.

Biphenyl itself has a 'twist angle' of about 45 degrees. If you put halogens in the 2 and 2' positions, it goes to ~ 60 (F) then to almost 90 degrees (Br and I).[Ref](https://pubs-acs-org/doi/pdf/10.1021/jp0122124) Since the Series 4 compounds generally have no ortho substituent on the RHS aryl substituent, and the biaryl bond links to a 5-membered ring, where one of the adjacent atoms is a nitrogen, I wouldn't be surprised if the twist angle is less than 45 degrees.

I think atropisomers would be a possibility if we had two ortho substituents on the RHS aryl group, ie. X,Y &ne; H. But, I think if we had only one ortho subsituent (eg. X = Cl, Y = H), we would still have relatively free rotation with the X group going past the triazole nitrogen.

**Comment by MFernflower** [comment]
I do agree that it is time to move away from the aminobenzene subunits -
especially if they have electron withdrawing groups attached to them!!! I
feel it is best to stick to indole like ring systems - As a matter of fact
I seem to remember proposing a benzofurazan compound but the boronic acid
for it was like 200 AUD

Thoughts on benzofurazan and other ring systems to try? @drc007 @mcoster

**Comment by drc007** [comment]
@maratsydney Are the crystal structure files for MK031-2a available? Would be interesting to look at the angle between the rings. @mcoster 

@MFernflower I think you mean benzofuran, might be interesting to compare a compound without an N-H but electron-rich heterocycles are often good substrates for CYP450 enzymes. Benzimidazole would also be interesting, as would the various azaindoles but some would require synthesis of the appropriate boronic acids.

**Comment by MFernflower** [comment]
The benzimidazole boronic acid can be grabbed from sigma
https://www.sigmaaldrich.com/catalog/product/aldrich/cds024320?lang=en&region=US
@drc007

**Comment by MFernflower** [comment]
Please ignore the previous comment I made @maratsydney @drc007 @david1597  - I had erroneously thought the indole ring was linked to the TP core at posistion C5!!!!  I was wrong - it's C6

Some relevant boronic acids linked at that position are commercially available: 

https://www.sigmaaldrich.com/catalog/product/aldrich/708631?lang=en&region=AU

https://www.sigmaaldrich.com/catalog/product/aldrich/702188?lang=en&region=AU (benzimidazole precursor - this will need to be subjected to Miyaura Borylation) - See this paper for interesting variant of the reaction https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3994125/

**Comment by maratsydney** [comment]
@drc007, I posted all xray files that I have with the structures in #46.

@MFernflower  benzimidazole is worth to try and I believe that two nitrogens in benzimidazole ring should be equivalent (and a real structure something average) - so it doesn't matter if it's position 5 or 6.  This is my opinion, would love to hear confirmation.

**Comment by MFernflower** [comment]
That might be a question @mcoster can answer

**Comment by mcoster** [comment]
I'm sure many can answer! They're tautomers and not quite equivalent, since one nitrogen is para-m and one meta- to the attachment point, but I suspect they would be very similar in energy and rapidly interconverting, and hence shouldn't make much difference whether it is 5- or 6- position.

**Comment by MFernflower** [comment]
@mcoster Believe it or not I actually did not realize until you mentioned it that benzimidazole tautomerizes

**Comment by MFernflower** [comment]
@maratsydney alongside the benzimidazole I would be curious to see what the homologous quinoline compound would fair in terms of potency since it has no N-H bond
